cli/src/virtual_appliance/bootstrap.sh

```diff
@@ -31,40 +31,41 @@ trap trap_handler ERR
 
 BASEDIR=$(dirname "${BASH_SOURCE}")
 
+provisioner pillar_export
```

Comment:
@ypise I guess it is outdated now?

Reply:
@andkononykhin2 yes, it's not required. I've updated it to:
`provisioner confstore_export` on line 42